app/config/config.yml

```diff
@@ -0,0 +1,119 @@
+imports:
+    - { resource: "@SWPBridgeBundle/Resources/config/app/config.yml" }
+    - { resource: "@SWPUserBundle/Resources/config/app/config.yml" }
+    - { resource: parameters.yml }
+    - { resource: security.yml }
+    - { resource: services.yml }
+
+# Put parameters here that don't need to change on each machine where the app is deployed
+# http://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
+parameters:
+    locale: en
+
+framework:
+    secret: '%secret%'
```

Comment:
let's use environment variables (as it is supported by Symfony >= 3.2) instead of simple parameters.